utils/template.go

```diff
@@ -73,10 +73,6 @@ func RenderTemplate(w http.ResponseWriter, context structs.PageContext, data int
 			}
 			return c
 		},
-		"showsToHours": func(shows []myradio.ShowMeta) int {
```

Comment:
would it be worth leaving the method in and just commenting out the html?

Reply:
It's not like it does much...

Reply:
fair